# patch to enable linux-x86 arch in GAMESS comp script
# includes a small fix that compiles module zheev using -O1
# since anything more agressive might break things

--- gamess-orig/comp	2005-07-23 20:30:18.000000000 +0000
+++ gamess/comp	2005-09-06 16:27:07.000000000 +0000
@@ -14,8 +14,8 @@
 #      linux-ia64,linux-pc,mac32,macG5,necsx,sgi32,sgi64,sun32,sun64
 #  2. Choose your GAMESS directory tree.
 #
-set TARGET=ibm64
-chdir /u1/mike/gamess
+set TARGET=linux-pc
+#chdir /u1/mike/gamess
 #
 #    ---- and now the script begins...
 #
@@ -992,11 +992,12 @@
 #   Pentium II fractionally).  None of these optimizations are used below.
 #
 if ($TARGET == linux-pc) then
-   set OPT = '-O2'
+   set gentoo-OPT = '-O2'
    if ($MODULE == ormas1) set OPT='-O2 -fno-move-all-movables' # RH9, g77 3.3
 #
 #   g77 does not support FORTRAN logical units above 99, or 128 bit math.
 #
+   if ($MODULE == zheev) set OPT='-O1' 
    if (($MODULE == qeigen) || ($MODULE == int2c)) then
       mv $MODULE.f $MODULE.junk
       sed -e "s/Q-/D-/g" \
@@ -1010,7 +1011,7 @@
       rm -f $MODULE.junk
    endif
    set echo
-   g77 -c $OPT -malign-double -fautomatic \
+   gentoo-g77 -c $OPT -malign-double -fautomatic \
            -Wno-globals -fno-globals $MODULE.f
    unset echo
 endif
